pwndbg/aglib/disasm/instruction.py

```diff
@@ -555,29 +554,81 @@ def __repr__(self) -> str:
         return f"[{info}]"
 
 
-def make_simple_instruction(address: int) -> PwndbgInstruction:
-    """
-    Instantiate a PwndbgInstruction for an architecture that Capstone/pwndbg doesn't support (as defined in the CapstoneArch structure)
-    """
-    ins = pwndbg.dbg.selected_inferior().disasm(address)
-    asm = ins["asm"].split(maxsplit=1)
+# Represents a disassembled instruction when Capstone does not support the given architecture
+class ManualPwndbgInstruction(PwndbgInstruction):
+    def __init__(self, address: int) -> None:
+        ins: DisassembledInstruction = pwndbg.dbg.selected_inferior().disasm(address)
+        asm = ins["asm"].split(maxsplit=1)
+
+        # ERROR: some of the functions depends on this attribute existing
+        # Give this object all the attributes it should have (defined in constructor)
+        self.address = address
+        self.size = ins["length"]
+
+        self.mnemonic = asm[0].strip()
+        self.op_str = asm[1].strip() if len(asm) > 1 else ""
```

Comment:
can `ins["asm"]` be splitted once and then not splitted here maybe?

Reply:
"asm" returns eg: `mov    r4, #0` so `strip` must be called